src/org/scada_lts/config/ScadaConfig.java

```diff
@@ -136,6 +136,10 @@
 
 	private Optional<Integer> optimizationLevelJs = Optional.empty();
 
+	public static final String DONT_CREATE_EVETS_FOR_EMAIL_ERROR = "abilit.DONT_CREATE_EVETS_FOR_EMAIL_ERROR";
```

Comment:
Is it possible to rename this variable , it means, by just put char 'N' :
DONT_CREATE_EVETS_FOR_EMAIL_ERROR - > DONT_CREATE_EVENTS_FOR_EMAIL_ERROR